src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj

```diff
@@ -1,29 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0-android</TargetFrameworks>
-    <ImplicitUsings>enable</ImplicitUsings>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
     <Nullable>annotations</Nullable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Remove="Windows\WinRT\windows.ui.notifications.mcidl" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <AdditionalFiles Include="Windows\WinRT\windows.ui.notifications.mcidl" />
-  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
+    <PackageReference Include="Tmds.DBus.Protocol" Version="0.19.0" />
```

Comment:
Those version of tmdbus are known to crash KDE and shouldn't be used with Avalonia codebase

Reply:
@kekekeks @jmacato was it reported anywhere?

Reply:
https://github.com/AvaloniaUI/Avalonia/pull/16129#issuecomment-2198590303

Reply:
Yeah, it's not an actionable report.

Reply:
https://github.com/AvaloniaUI/Avalonia/issues/15655